framework/src/auxkernels/AuxKernel.C

```diff
@@ -194,20 +199,24 @@ AuxKernel::compute()
       _local_ke.zero();
 
       // assemble the local mass matrix and the load
-      for (unsigned int i = 0; i < _test.size(); i++)
-        for (_qp = 0; _qp < _qrule->n_points(); _qp++)
+      for (_qp = 0; _qp < _qrule->n_points(); _qp++)
```

Comment:
This is a decent idea... but realize that it may actually make this slower if `computeValue()` is simple.

The reason it may be slower is that you're now moving through the first index in `_test[i][_qp]` fastest... so you're skipping through memory which will be slower.

Have you done any timing to show that this is better / worse?  I would be interested in seeing the tradeoffs.